spec/support/vcr_cassettes/bgs/benefit_claim/update_5103_200.yml

```diff
@@ -264,668 +273,17 @@ http_interactions:
       - chunked
       Content-Type:
       - text/xml; charset=utf-8
+      Strict-Transport-Security:
+      - max-age=16000000; includeSubDomains; preload;
     body:
       encoding: UTF-8
       string: <?xml version='1.0' encoding='UTF-8'?><S:Envelope xmlns:env="http://schemas.xmlsoap.org/soap/envelope/"
         xmlns:S="http://schemas.xmlsoap.org/soap/envelope/"><env:Header><work:WorkContext
-        xmlns:work="http://oracle.com/weblogic/soap/workarea/">rO0ABXdUAB13ZWJsb2dpYy5hcHAuQ29ycG9yYXRlRGF0YUVBUgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMi4zMQAA</work:WorkContext></env:Header><S:Body><ns0:findBenefitClaimDetailsByBnftClaimIdResponse
+        xmlns:work="http://oracle.com/weblogic/soap/workarea/">rO0ABXdUAB13ZWJsb2dpYy5hcHAuQ29ycG9yYXRlRGF0YUVBUgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMy4xNAAA</work:WorkContext></env:Header><S:Body><ns0:findBenefitClaimDetailsByBnftClaimIdResponse
         xmlns:ns0="http://claimstatus.services.ebenefits.vba.va.gov/"><BenefitClaimDetailsDTO><attentionNeeded>No</attentionNeeded><baseEndPrdctTypeCd>930</baseEndPrdctTypeCd><benefitClaimId>600131328</benefitClaimId><bnftClaimLcStatus><phaseChngdDt>2022-09-21T09:51:20</phaseChngdDt><phaseType>Complete</phaseType><phaseTypeChangeInd>78</phaseTypeChangeInd></bnftClaimLcStatus><bnftClaimTypeCd>930AC</bnftClaimTypeCd><claimCompleteDt>2022-09-21T09:51:20</claimCompleteDt><claimDt>2018-06-19</claimDt><claimStatus>CLR</claimStatus><claimStatusType>Appeal</claimStatusType><decisionNotificationSent>Yes</decisionNotificationSent><developmentLetterSent>No</developmentLetterSent><endPrdctTypeCd>930</endPrdctTypeCd><filed5103WaiverInd>Y</filed5103WaiverInd><poa>DISTRICT
         OF COLUMBIA, OFFICE OF VETERANS AFFAIRS</poa><programType>CPL</programType><ptcpntClmantId>600043201</ptcpntClmantId><ptcpntVetId>600043201</ptcpntVetId><regionalOfficeJrsdctn>St.
         Petersburg</regionalOfficeJrsdctn><tempRegionalOfficeJrsdctn>Montgomery</tempRegionalOfficeJrsdctn><wsyswwn><addressLine1>Regional
         Office</addressLine1><addressLine2>PO Box 1437</addressLine2><addressLine3></addressLine3><city>St.
         Petersburg</city><state>FL</state><zip>33731</zip></wsyswwn></BenefitClaimDetailsDTO></ns0:findBenefitClaimDetailsByBnftClaimIdResponse></S:Body></S:Envelope>
-  recorded_at: Thu, 10 Nov 2022 16:10:47 GMT
-- request:
```

Comment:
I presume the next 2 requests aren't needed since tests pass, so this should be fine.